packages/playground/src/explorer/farms.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <view-layout>
+    <v-row>
+      <v-col>
+        <node-filters v-model="filterInputs" v-model:valid="isValidForm" @update:model-value="inputFiltersReset" />
+        <FarmsTable :items="farms" :loading="loading" v-model:selectedFarm="selectedFarm" @open-dialog="openDialog" />
+      </v-col>
+    </v-row>
+    <farmDialog v-if="selectedFarm" :openDialog="isDialogOpened" :farm="selectedFarm" @close-dialog="closeDialog" />
+  </view-layout>
+</template>
+
+<script lang="ts" setup>
+import type { CertificationType, FarmsQuery, PublicIp } from "@threefold/gridproxy_client";
+import { onMounted, ref, watch } from "vue";
+import { useRoute } from "vue-router";
+
+import router from "@/router";
+import { inputsInitializer } from "@/utils/filter_farms";
+
+import { getFarmQueries, getFarms } from "./utils/helpers";
+import {
+  type FarmFilterInputs,
+  type FarmFilterOptions,
+  type FarmMixedFilter,
+  farmOptionsInitializer,
+} from "./utils/types";
+
+export interface IFarm {
+  farmId: number;
+  name: string;
+  twinId: number;
+  pricingPolicyId: number;
+  certificationType: CertificationType;
+  publicIps: PublicIp[];
+  stellarAddress?: string;
+  totalPublicIp: number;
+  usedPublicIp: number;
+  freePublicIp: number;
+}
+const route = useRoute();
+
+const loading = ref<boolean>(false);
+const farms = ref<IFarm[]>([]);
+const isDialogOpened = ref<boolean>(false);
+const selectedFarm = ref<IFarm>();
+const filterInputs = ref<FarmFilterInputs>(inputsInitializer);
+const filterOptions = ref<FarmFilterOptions>(farmOptionsInitializer);
+const isValidForm = ref<boolean>(false);
+const farmsCount = ref<number>(0);
+const mixedFilters = ref<FarmMixedFilter>({
```

Comment:
You don't need this mixed filter, you only have inputs.

The `mixedFilter` was created in the `node explorer` because we have two types of filters
1 - input filters which represent the `form-filters`
2 - options filter which represents the options like `node status`, `GPU support`, and `gateway support`

but in your case, you only need to create the inputs filter, see the `dedicated-nodes` component for more details.